### PR TITLE
Make client keepAliveInterval configurable in client side

### PR DESCRIPTION
**Pull request by zhaijack**
### Motivation

Currently `keepAliveInterval` is hard coded as 30s in ClientCnx, Would like to make it configurable.

### Modifications

Add `keepAliveIntervalSeconds` in ClientConfigurationData, to make it configurable.

### Result

user could config client `keepAliveIntervalSeconds`

### CODE REVIEW

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java [hunk 1]
```diff
@@ -53,6 +53,7 @@
     private boolean tlsHostnameVerificationEnable = false;
     private int concurrentLookupRequest = 50000;
     private int maxNumberOfRejectedRequestPerConnection = 50;
+    private int keepAliveIntervalSeconds = 30;
```
Comment (merlimat) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java]: To make it accessible, this should be exposed in `ClientBuilder` interface as well

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java [hunk 2]
```diff
@@ -107,7 +107,7 @@
     }
 
     public ClientCnx(ClientConfigurationData conf, EventLoopGroup eventLoopGroup) {
-        super(30, TimeUnit.SECONDS);
+        super(conf.getKeepAliveIntervalSeconds(), TimeUnit.SECONDS);
```
Comment (merlimat) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java]: This is changing the client side timeout to timeout a connection with a non-responsive broker. The broker timeout, though, is still set at 30seconds, so broker might still close the connection based on that timeout

**Comment by zhaijack** [comment]
retest this please

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java [hunk 3]
```diff
@@ -155,6 +155,12 @@ public ClientBuilder maxNumberOfRejectedRequestPerConnection(int maxNumberOfReje
         return this;
     }
 
+    @Override
+    public ClientBuilder keepAliveIntervalSeconds(int keepAliveIntervalSeconds) {
```
Comment (merlimat) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java]: For consistency, in most other config options with times we're taking the interval and the time unit, then in the `ClientConfigurationData` we convert into a specific unit (which for this case seconds are perfectly fine).

**Comment by sijie** [comment]
retest this please

**Pull request merged**